src/models/playlist.js

```diff
@@ -267,7 +267,8 @@ const updatePlaylistField = async (request, reply) => {
         const { ok, n } = response;
 
         if (ok) {
-            return reply({ success: true, message: `${field} was updated` });
+            let updateMessage = (field === 'playlistDate') ? 'The playlist date was updated' : `The document field '${field}' was updated`;
+            return reply({ success: true, message: updateMessage });
```

Comment:
rather than use some conditional logic here to return a special message in this one case -- what happens when we need more special messages for more cases? -- let's create a mapping from fields to messages instead